dvc/dependency/repo.py

```diff
@@ -49,10 +49,10 @@ def _make_repo(self, **overrides):
 
     def status(self):
         with self._make_repo() as repo:
-            current = repo.find_out_by_relpath(self.def_path).info
+            current = repo.find_out_by_path(self.def_path).info
```

Comment:
Not sure why this was required?

Reply:
Technically its not required, though considering change in `find_outs_by_path`, now `find_out_by_relpath` will  work both with abs and relpath, thats why I decided to rename it.

Reply:
@pared yeah, but why do we need to make `find_out_by_path` accept relpaths?

Reply:
To pass original path to OutputNotFoundError. It makes sense because it passes originally inputed path to error, which is easier to understand, since its user input. Calculating relpaths/abspaths was the original reason for ambiguous paths flooding `stderr` when output could not be found.